pkg/pipeline/extract/conntrack/conntrack.go

```diff
@@ -143,6 +71,7 @@ func (ct *conntrackImpl) Extract(flowLogs []config.GenericMap) []config.GenericM
 				KeysFrom(fl, ct.config.KeyDefinition).
 				Aggregators(ct.aggregators).
 				Build()
+			conn.setNextUpdateReportTime(ct.clock.Now().Add(ct.config.UpdateConnectionInterval.Duration))
```

Comment:
Do you want to add this as another `build` step instead of explicitaly calling the function

Reply:
Done